src/styles/variables.scss

```diff
@@ -0,0 +1,13 @@
+$cell-size: 100px;
+$col-count: 7;
+$gap-size: 1px;
```

Comment:
Consider using a more descriptive variable name like `$calendar-gap-size` to improve readability.